### PR TITLE
Update for .NET 6 and VS 2022 RTM 

### PR DESCRIPTION
**Pull request by jeromelaban**
## PR Type

What kind of change does this PR introduce?
- Build or CI related changes

## What is the new behavior?

- Add Xamarin templates in VS 2022 VSIX
- Update build targets to .NET 6 RTM

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: src/Uno.UI/UI/Xaml/Controls/ItemsStackPanel/ItemsStackPanelLayout.iOS.cs [hunk 1]
```diff
@@ -6,11 +6,15 @@
...
+#if NET6_0_OR_GREATER
+using ObjCRuntime;
+#endif
+
 #if XAMARIN_IOS_UNIFIED
 using Foundation;
 using UIKit;
 using CoreGraphics;
-using LayoutInfo = System.Collections.Generic.Dictionary<Foundation.NSIndexPath, UIKit.UICollectionViewLayoutAttributes>;
+using LayoutInfo = System.Collections.Generic.Dictionary<Foundation.NSIndexPath, UIKit.UICollectionViewLayoutAttributes>;using ObjCRuntime;
```
Comment (davidjohnoliver) [inline src/Uno.UI/UI/Xaml/Controls/ItemsStackPanel/ItemsStackPanelLayout.iOS.cs:17]: typo?

File: src/Uno.UI/UI/Xaml/Controls/Picker/Picker.iOS.cs [hunk 2]
```diff
@@ -13,6 +13,7 @@
 using System.Windows.Input;
 using CoreGraphics;
 using Uno.Extensions.Specialized;
+using ObjCRuntime;
```
Comment (davidjohnoliver) [inline src/Uno.UI/UI/Xaml/Controls/Picker/Picker.iOS.cs:16]: missing conditional?

File: src/Uno.UI/UI/Xaml/Controls/TextBlock/TextBlock.iOS.cs [hunk 3]
```diff
@@ -11,6 +11,7 @@
 using Windows.UI.Text;
 using Uno.UI;
 using Windows.UI;
+using ObjCRuntime;
```
Comment (davidjohnoliver) [inline src/Uno.UI/UI/Xaml/Controls/TextBlock/TextBlock.iOS.cs:14]: missing conditional?

**Review by davidjohnoliver — APPROVED** [review]

**Review by MartinZikmund — APPROVED** [review]

File: src/Uno.UI/UI/Xaml/Controls/ItemsStackPanel/ItemsStackPanelLayout.iOS.cs (hunk 1 above)
Comment (jeromelaban) [reply src/Uno.UI/UI/Xaml/Controls/ItemsStackPanel/ItemsStackPanelLayout.iOS.cs]: A big one :)

**Pull request merged**